Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+	private void OnTriggerStay2D(Collider2D collision)
+	{
+		if (collision.CompareTag("Player"))
+		{
+			if (Input.GetButtonDown("Activate"))
+			{
+
+				Debug.Log("player has door activated");
+			}
+		}
+	}
+
+	public string sceneToLoad;
```

Comment:
Weirdly indented string
String is public - does it need to be public?
Variable shouldn't be declared in between methods